content/automate/ManageIQ/System/Event/RequestEvent/Request.class/request_completed_with_success.yaml

```diff
@@ -0,0 +1,12 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: request_completed_with_success
+    inherits: 
+    description: 
+  fields:
+  - rel5:
+      value: "/System/Policy/request_completed_with_success"
```

Comment:
@AparnaKarve 
/System/Policy/request_completed_with_success is not defined in https://github.com/ManageIQ/manageiq-content/tree/master/content/automate/ManageIQ/System/Policy.class are you planning on using the missing instance?

Reply:
same question as above https://github.com/ManageIQ/manageiq-content/pull/429#discussion_r219965042